tgbot/src/api/auth.py

```diff
@@ -0,0 +1,98 @@
+import typing as tp
+
+from collections import namedtuple
+
+from aiohttp import ClientResponse
+
+from api.client import client
+
+# ------------------------------------------------------------------------------
+
+Session = namedtuple("Session", ["id", "code", "chat_id"])
+
+
+async def fetch_session(params: tp.Dict[str, tp.Any]) -> tp.Optional[Session]:
+    """Fetch session for chat. Session may not exist, hence `Optional`."""
+
```

Comment:
I would like to name this function `get_session`, but I'm still ok with `fetch` version. Is there some sort of a naming rule that I do not know?

Reply:
`fetch` implies network access.

Reply:
Ok